packages/autocomplete-js/src/utils/mergeClassNames.ts

```diff
@@ -8,7 +8,8 @@ export function mergeClassNames(
       const accValue = acc[key];
       const currentValue = current[key];
 
-      acc[key] = [accValue, currentValue].filter(Boolean).join(' ');
+      if (accValue !== currentValue)
+        acc[key] = [accValue, currentValue].filter(Boolean).join(' ');
```

Comment:
We'll enable the [`curly`](https://eslint.org/docs/rules/curly) ESLint rule to automate this.

```suggestion
      if (accValue !== currentValue) {
        acc[key] = [accValue, currentValue].filter(Boolean).join(' ');
      }
```

Reply:
I was wondering if it was on purpose but good to know!